### DISCUSSION TITLE
How to pass Database notification across multiple PanelProvider in filament3?

### DISCUSSION THREAD

**Discussion by CodeWithPranta** [filamentphp/filament#8986] [General]
I'm developing a school management system in filament. I have default AdminPanelProvider with other 4 panel providers for students, teachers and guardians. I have implemented database notification in AdminPanelProvider with databaseNotifications() method. I want if admin create a new notice using NoticeResource teachers, students and guardians will get notification like AdminPanelProvider or Admin Dashboard.

**Comment by programmingChiira** [comment]
Hi there, did you get a solution for this yet?

**Reply by CodeWithPranta** [reply]
Yes, read their documentation https://filamentphp.com/docs/3.x/notifications/database-notifications
